### PR TITLE
timer: remove update_status job

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
timer: remove update_status job

Since there is no longer a mismatch between root and non-root status, we no longer need that job around

## Test Steps
Run the modified tests and certify that the update_status job is not referenced anymore by the cli commands and the timer job

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [x] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

File: features/attached_commands.feature [hunk 1]
```diff
@@ -729,6 +729,7 @@ Feature: Command behaviour when attached to an Ubuntu Pro subscription
            | focal   | cloud-init-dev-ubuntu-daily-focal  |
            | jammy   | cloud-init-dev-ubuntu-daily-jammy  |
 
+    @wip
```
Comment (orndorffgrant) [inline features/attached_commands.feature:732]: Looks like this was leftover

**Review by orndorffgrant — APPROVED** [review]
LGTM! just a leftover `@wip`

**Comment by CalvoM** [comment]
I see the [docs](https://github.com/canonical/ubuntu-advantage-client/blob/5f18ceb77f23fa9ca1ed421671923c98084a8e82/docs/explanations/what_are_the_timer_jobs.md?plain=1#L14) have a remaining reference to `update_status` timer.

**Comment by lucasmoura** [comment]
@CalvoM good catch. I have updated the PR

**Review by orndorffgrant — APPROVED** [review]
LGTM! The failing test is fixed in `main`

**Comment by orndorffgrant** [comment]
There are some conflicts now, and a rebase on main should also fix the CI failure, then it should be good to land.

**Review by renanrodrigo — APPROVED** [review]
+1 rebase and :shipit:

**Comment by lucasmoura** [comment]
@orndorffgrant @renanrodrigo updated

**Comment by orndorffgrant** [comment]
fails look like flakes, rerunning...

**Pull request merged**